packages/host/src/lib/host.ts

```diff
@@ -1054,6 +1055,12 @@ export class Host implements IComponent {
             this.telemetryAdapter = await getTelemetryAdapter(this.config.telemetry.adapter, this.config.telemetry);
             this.telemetryAdapter.logger.pipe(this.logger);
 
+            let ipAddress = require("ext-ip")();
+            ipAddress.on("ip", (ip: any) => {
+                this.ipvAddress = ip;
+            });
+            await ipAddress();
```

Comment:
@alicja-gruzdz @daro1337 @stokowski - I'm not sure about that, wouldn't we know what was the reporting IP anyway? And this module uses 3rd party services to get the IP...

Reply:
Perhaps Loki has something, @Tatarinho is looking into it. 
Previously, @daro1337 suggested https://www.npmjs.com/package/public-ip but there were issues with it. It also uses external services and has deps.

Reply:
There was an import issue with the library https://www.npmjs.com/package/public-ip [ERR_REQUIRE_ESM]. I'd merge this PR with the `ext-ip`, and then replace it if required.